apps/sensenet/src/components/content/Explore.tsx

```diff
@@ -112,6 +113,7 @@ export function Explore({
         if (contentTypeName) {
           return (
             <NewView
+              key={activeContent}
```

Comment:
activeContent is not given in this case, and can not switch from one new view to another, so it can be omitted here